src/mono/mono/metadata/mono-debug.c

```diff
@@ -1097,13 +1097,31 @@ mono_register_symfile_for_assembly (const char *assembly_name, const mono_byte *
 	bundled_symfiles = bsymfile;
 }
 
+static gboolean
+bsymfile_match (BundledSymfile *bsymfile, const char *assembly_name)
+{
+	if (!strcmp (bsymfile->aname, assembly_name))
+		return TRUE;
+#ifdef ENABLE_WEBCIL
+	const char *p = strstr (assembly_name, ".webcil");
+	/* if assembly_name ends with .webcil, check if aname matches, with a .dll extension instead */
+	if (p && *(p + 7) == 0) {
```

Comment:
```suggestion
	if (p && *(p + strlen(".webcil") == 0) {
```
the compiler will do the right thing